src/shared/components/post/post.tsx

```diff
@@ -258,8 +285,12 @@ export class Post extends Component<PostRouteProps, PostState> {
     isIsomorphic: false,
     notifications: "replies_and_mentions",
     editLoading: false,
+    highlightedCommentId: null, // Start with post highlighted
+    imageExpanded: false, // Start with image not expanded
```

Comment:
Remove.

If you need to show the expanded image, use `post_listing_mode: card` vs `small_card`